scripts/json_config_expand_relpaths.js

```diff
@@ -97,7 +97,7 @@ var config = json.parse(fs.readFileSync(inputfile, {encoding: 'utf8'}).toString(
 // Expand relative paths
 
 if (typeof String.prototype.startsWith !== 'function') {
```

Comment:
I think it should be fine to remove this whole block. ``String.prototype.startsWith`` is part of [ECMAScript6, and would only be unavailable on IE < 12](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith). Since this is an internal utility script, and not compiled into the web client, I'd think compatibility fixes should be unnecessary.

Ping @jcfr , since he added it.